### DISCUSSION TITLE
my case its not working idk what its wrong , i did a video

### DISCUSSION THREAD

**Discussion by Toei79** [GerryDazoo/Slinger#379] [General]
my model is 350.

so i did a video, where you can see command line , executing  .py file,  the console.log showing the id, and password also showing the .ini file showing its all in order for what i understand,

previously i started sling player , then i check the console log.

https://user-images.githubusercontent.com/79808610/204681163-f8fd043e-1062-43a6-92f8-91754c7bf7b5.mp4

![11-29-2022 (195453)](https://user-images.githubusercontent.com/79808610/204681216-8c8480da-621a-4bcd-b33e-fcc9cf778ec8.jpg)

so i check one 350 that i reset finally i get video so basically i only can get component on that one because i cant switch to composite without remote, or its any possible ?

**Comment by socelillyhammer** [comment]
Send your config file please

**Comment by Toei79** [comment]
[config.txt](https://github.com/GerryDazoo/Slinger/files/10118585/config.txt)


yeah look , thanks

**Reply by socelillyhammer** [reply]
Try this see attached. I cleaned you up a bit.  I enabled your remote to be the basic remote.

So, if you IP is correct, your VLC should pickup the video as your port is 8080.

If you see the video on the VLC then open firefox (for example) and you type in http://ipaddress:8080/Remote to bring up your remote

If you still have problems, what is the directory where your stuff is all located in? for example mine is in 
C:\Users\username\Downloads\Slinger-main\Slinger-main

And we will go from there to check out your RunSling.bat file.

[config.txt](https://github.com/GerryDazoo/Slinger/files/10118642/config.txt)




![Screenshot 2022-11-29 205028](https://user-images.githubusercontent.com/119097385/204688167-d89b00c7-8803-4b49-9051-738596a1878d.jpg)

**Reply by Toei79** [reply]
idk im using linux 
my path its this one.  i dont use bat file in linux for what i understand.

/home/ubuntu/Slinger

**Reply by socelillyhammer** [reply]
so your RunSling.bat file should read "cd C:\home\ubuntu\Slinger"

meaning you renamed your folder Slinger

and C is obviously whatever your C drive is labeled as C or D or E etc

Save that and then run everything and your should be good.

**Reply by Toei79** [reply]
thats windows im in linux , idk if that works.

in the other one that i did hard reset was working fine

**Reply by socelillyhammer** [reply]
correct so there are notes in the .bat file as to how to direct it and label everything.  if that doesnt work ask @GerryDazoo

**Reply by Toei79** [reply]
sadly its not working its a setup process for linux i guess that i did already

## Linux/RaspberryPi Notes README.md

**Comment by abarberenaCPDS** [comment]
Hello @Toei79 👋 ,

Based on your video, the error message is clear, `Check your admin password`, your password, *E1:nnnnnn* seems to be other than the admin password, maybe it's the user?

Just to confirm, what's the source of your `console.log` file?

**Reply by Toei79** [reply]
this one
AppData\Roaming\Sling Media\Slingplayer Desktop\Logs

and its from today.  using the app then checking log file

**Reply by Toei79** [reply]
> Hello @Toei79 👋 ,
> 
> Based on your video, the error message is clear, `Check your admin password`, your password, _E1:nnnnnn_ seems to be other than the admin password, maybe it's the user?
> 
> Just to confirm, what's the source of your `console.log` file?

you right. i found the password, not encrypted, wondering if that helps in the log
